spec/controllers/api/projects_controller_spec.rb

```diff
@@ -379,4 +379,39 @@ def project_response(project)
       put :external_auth, params: { id: external_auth.project_id }, format: :json
     end
   end
+
+  describe '#work_times' do
```

Comment:
please add more specs for all other roles (manager, regular & leader). To make sure we don't have any security vulnerability here.